tests/ci_scripts/run_premerge_gpu.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+#
+# Copyright (c) 2020-2022, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -ex
+
+BUILD_TYPE=all
```

Comment:
currently we do not build_type here until developers require complicated test combination support

could just remove line 20-29